generators/client/templates/angular/src/main/webapp/app/admin/audits/audits.service.ts.ejs

```diff
@@ -28,10 +28,8 @@ import { Audit } from './audit.model';
 export class AuditsService  {
     constructor(private http: HttpClient) { }
 
-    query(req: any): Observable<HttpResponse<Audit[]>> {
+    query(req: { page?: number; size?: number; sort?: string[]; fromDate: string; toDate: string }): Observable<HttpResponse<Audit[]>> {
         const params: HttpParams = createRequestOption(req);
-        params.set('fromDate', req.fromDate);
-        params.set('toDate', req.toDate);
```

Comment:
Why are we removing thos params from the request?

Reply:
Those parameters are not removed from the request. These parameters are already assigned in the first line of this function and no need to assign values again.